sites/partners/src/pages/listings/[id]/lottery.tsx

```diff
@@ -234,35 +241,67 @@ const Lottery = (props: { listing: Listing }) => {
             <ListingStatusBar status={listing?.status} />
             <section className={styles["lottery"]}>
               <div className={styles["parent"]}>
-                <div className={styles["container"]}>
-                  <div className={styles["main"]}>
-                    <Card spacing={"lg"}>{getMainContent()}</Card>
-                  </div>
-                  <aside className={styles["side"]}>
-                    <>
-                      {getActions()}
-                      <Card>
-                        <CardHeader>
-                          <Heading priority={2} size={"lg"}>
-                            {t("listings.lottery.history")}
-                          </Heading>
-                        </CardHeader>
+                {shouldExpireData && listing?.lotteryStatus === LotteryStatusEnum.expired ? (
```

Comment:
If formattedExpiryDate / expiryDate in general is optional, should this also check that that is not null?

Reply:
yep! I was just testing out passing the env var in the payload. Now it's all setup and ready for review again